java/core/src/main/java/co/worklytics/psoxy/storage/impl/ColumnarBulkDataSanitizerImpl.java

```diff
@@ -184,13 +184,8 @@ public void sanitize(@NonNull Reader reader,
         // by pseudonymizing IF column should happen to exist
         Set<String> outputColumnsCI = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
         outputColumnsCI.addAll(applyReplacements(headersCI, columnsToRename));
-        Sets.SetView<String> missingColumnsToPseudonymize =
-            Sets.difference(columnsToPseudonymize, outputColumnsCI);
-        if (!missingColumnsToPseudonymize.isEmpty()) {
-            log.info(String.format("Columns to pseudonymize (%s) missing from set found in file (%s)",
-                "\"" + String.join("\",\"", missingColumnsToPseudonymize) + "\"",
-                "\"" + String.join("\",\"", headersCI) + "\""));
-        }
```

Comment:
never figured out exactly what was wrong with this. But pointing customer demo config, where problem had been reproduced, at this branch solved the issue.